iwyu_preprocessor.h

```diff
@@ -202,13 +203,13 @@ class IwyuPreprocessorInfo : public clang::PPCallbacks,
 
   void FileChanged(clang::SourceLocation loc, FileChangeReason reason,
                    clang::SrcMgr::CharacteristicKind file_type,
-                   clang::FileID PrevFID) override;
+                   clang::FileID exiting_from) override;
   void FileSkipped(const clang::FileEntry& file, const clang::Token &filename,
                    clang::SrcMgr::CharacteristicKind file_type) override;
   // FileChanged is actually a multi-plexer for 4 different callbacks.
   void FileChanged_EnterFile(clang::SourceLocation file_beginning);
   void FileChanged_ExitToFile(clang::SourceLocation include_loc,
-                              clang::FileID exiting_from);
+                              clang::FileID exiting_from_id);
```

Comment:
Can we please make names consistent? So it is either `exiting_from` or `exiting_from_id` in both places.